docs/source/konieczny.rst

```diff
@@ -0,0 +1,37 @@
+.. Copyright (c) 2020, F. L. Smith
+
+   Distributed under the terms of the GPL license version 3.
+
+   The full license is in the file LICENSE, distributed with this software.
+
+Konieczny
+=========
+
+``libsemigroups`` contains a preliminary implementation of Konieczny's
+generalisation of Lallement-McFadden's algorithm for computing the structure of
```

Comment:
Should this be "Lallement and McFadden's algorithm" or "the Lallement-McFadden algorithm"?